rules/default.js

```diff
@@ -78,20 +74,21 @@ module.exports = function (options) {
 
     list : function (body, ordered) {
         var result = {
-            elem    : 'list',
```

Comment:
maybe add:
```
mix: {
    elem: 'list',
}
```
for writing common css for all lists, for example — common font-family or padding